pages.de/android/input.md

```diff
@@ -0,0 +1,26 @@
+# input
+
+> Sende Eventcodes oder Touchscreengesten an ein Android Gerät.
+> Eventcodes sind die Eingabe eines Zeichens, Beenden eines Anrufs, Erhöhen der Lautstärke, usw.
+> Dieser Befehl kann nur mit `adb shell` verwendet werden.
+> Mehr Informationen: <https://developer.android.com/reference/android/view/KeyEvent.html#constants_1>.
```

Comment:
There are 3 description lines here (excluding the link), but the [English page](https://github.com/tldr-pages/tldr/blame/4c059df5a66283a0a7c85d653af978b21342ea06/pages/android/input.md#L3-L6) only has two (besides the link).  Is this a mistake?

Also, I'm not sure but I don't think we're supposed to have more than two description lines (excluding the link, of course) 🤔

Reply:
Now I see that the English page is waiting for this change as well: https://github.com/tldr-pages/tldr/pull/5807/files#diff-6d7b5ac94fb71d5f0e18bbe601e0380866cd931813bd86356e2f3e722ae99fd4R5